sdk/storage/azure-storage-blob/tests/test_container.py

```diff
@@ -1082,11 +1116,14 @@ def test_standard_blob_tier_set_tier_api_batch(self):
             assert not blob_ref2.blob_tier_inferred
             assert blob_ref2.blob_tier_change_time is not None
 
+        try:
             response = container.delete_blobs(
                 'blob1',
                 'blob2',
                 'blob3',
             )
+        except:
+            pass
```

Comment:
Why are we passing all exceptions here? The test could be failing

Reply:
removed the block..it just crept in.